modules/platforms/cpp/core/src/impl/cluster/cluster_node_impl.cpp

```diff
@@ -139,8 +144,17 @@ namespace ignite
 
             void ClusterNodeImpl::ReadConsistentId(BinaryReaderImpl& reader)
             {
-                consistentIdOffset = reader.GetStream()->Position();
-                reader.Skip();
+                int8_t typeId = reader.ReadInt8();
```

Comment:
Not sure about this part, need to consult with other team members.

Reply:
Ok, let's revert this part and leave it the way it was before with `T` as a return type.
We need to mention in doxycomment that be default `T` should be `std::string` though.

Reply:
It was decided  to leave not templated function which will return std:string after discussion.